Developers/Newcomers_Guide.rst

```diff
@@ -46,7 +46,7 @@ Step 3. Picking Up an Issue
 ---------------------------
 
```

Comment:
first of all the shortlog is wrong. you are not changing the new comers link, it should be "Change newcomers' issues link" or so. 

also, in the commit body you should use present tense: Changes, not Changed, and it should explain why: "Changes the link because the organization was renamed, and the old link did not reflect the new changes, so it would lead to an empty page."

this should be all :)